imports/startup/client/routes.js

```diff
@@ -102,7 +102,7 @@ FlowRouter.route('/playlists', {
   }
 });
 
-FlowRouter.route('/playlists/:id', {
+FlowRouter.route('/playlist/:id', {
```

Comment:
I've corrected this in my last commit. The singularity/plurality thing bugged me a bit, so I changed it.